FML/COLASolver/frontend/horndeski_parameters.ini

```diff
@@ -0,0 +1,73 @@
+#### Give the model its name
+model_name = 'ESS-A'
+# Here quantities related to objects in reduced Horndeski action are defined
+##################
+# Mass ratios #
+#################
+# If True, all mass ratios are set to 1 and values below are ignored
+set_all_to_one = True
+
+M_pG4 = 1.
+M_KG4 = 1.
+M_G3s = 1. 
+M_sG4 = 1.
+M_G3G4 = 1.
+M_Ks = 1.
+M_gp = 1.
+
+##################
+# Theory symbols #
+##################
+# Here you declare the symbols needed to construct your specific theory.
+# Typically this involves giving names to the constants in the model.
+
+#Syntax: left side is the variable name that python will call it by, right is a
+# string that indicates how the symbol should be displayed (use LaTeX notation)
+# the variable name (key) MUST end in "_parameter" (without quotation marks)
+#k1, g31, g32, k2 = sym.symbols("k_1 g_{31} g_{32} k_2")
+k1_symbol = "k_1"
+k2_symbol =  "k_2"
+g31_symbol = "g_31"
+g32_symbol = "g_32"
+
+
+#######################
+# Horndeski functions #
+#######################
+
+# See documentation for meaning of these functions
+
+K = k_1*X + k_2*X*X
+
+
+G3 = g_31*X + g_32*X*X
+
+
+G4 = 0.5 
```

Comment:
Think this section should be closer to the top of the file, just below the model name variable.